### PR TITLE
Add rajpra.html file to repo

### PR DESCRIPTION
**Pull request by rajpranz**
Added rajpra.html

### CODE REVIEW

**Comment by TheCandiko** [comment]
Congratulations! You've made your first Pull Request in Glorious Geckos!

This is your first comment. This comments are general about Pull Requests, we can also make more specific comments on files that you have changed. Even specific line of codes!

Please check out the README.md and continue with: [**6. Make a Comment for a Pull Request**](https://github.com/DarisCalinor/practice-for-geckos#6-make-a-comment-for-a-pull-request)

File: rajpra.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Chingu Voyage-4</title>
```
Comment (TheCandiko) [inline rajpra.html]: We should add Slack team name '_geckos-team-20_' into `<title>` tag.

**Review by shalinidc — APPROVED** [review]

File: rajpra.html (hunk 1 above)
Comment (rajpranz) [reply rajpra.html]: Yea thats a good idea

**Comment by hiralmakwana30** [comment]
Testing Waffle

**Pull request merged**